### PR TITLE
PMM-13159 Add release note

### PR DESCRIPTION
**Pull request by YashSartanpara1**
PMM 2.43.0 - Release Notes - Draft

List of tickets:
https://perconadev.atlassian.net/browse/PMM-13159

### CODE REVIEW

File: docs/release-notes/2.43.0.md [hunk 1]
```diff
@@ -28,6 +28,8 @@ This release introduces this and that
 
 Additionally, the endpoint now includes a customizable line-count parameter in the download URL, allowing you to specify a custom number of log lines or opt for unlimited log size. For more information, see the [API documentation](https://percona-pmm.readme.io/reference/logs).
 
+- [PMM-13159](https://perconadev.atlassian.net/browse/PMM-13159) - Removed experimental mongodb collection details dashboard as the same info is now available on the new experimental mongodb cluster summary dashboard.
```
Comment (ademidoff) [inline docs/release-notes/2.43.0.md:31]: I wonder if we should spell it as `MongoDB`?

Comment (YashSartanpara1) [reply docs/release-notes/2.43.0.md:31]: Agree, just checked older notes, dashboard names are written as MongoDB Collection Details. ill update the same

File: docs/release-notes/2.43.0.md [hunk 2]
```diff
@@ -28,6 +28,8 @@ This release introduces this and that
 
 Additionally, the endpoint now includes a customizable line-count parameter in the download URL, allowing you to specify a custom number of log lines or opt for unlimited log size. For more information, see the [API documentation](https://percona-pmm.readme.io/reference/logs).
 
+- [PMM-13159](https://perconadev.atlassian.net/browse/PMM-13159) - Removed experimental MongoDB Collection Details dashboard as the same info is now available on the new experimental MongoDB Cluster Summary dashboard.
```
Comment (catalinaadam) [inline docs/release-notes/2.43.0.md:31]:
```suggestion
- [PMM-13159](https://perconadev.atlassian.net/browse/PMM-13159) - Removed experimental MongoDB Collection Details dashboard as the same info is now available on the new experimental MongoDB Cluster Summary dashboard.
```
```suggestion
- [PMM-13159](https://perconadev.atlassian.net/browse/PMM-13159) - We have removed the experimental MongoDB **Collection Details** dashboard from PMM. You can now find up-to-date collection information on the new Mongo **Cluster Summary** dashboard, which provides a more comprehensive and accurate view of MongoDB collection metrics. 

If you have been using the **Collection Details** dashboard, make sure to now check the new **MongoDB Cluster Summary** dashboard for similar information.
```

**Review by BupycHuk — APPROVED** [review]

**Review by idoqo — APPROVED** [review]

File: docs/release-notes/2.43.0.md (hunk 2 above)
Comment (YashSartanpara1) [reply docs/release-notes/2.43.0.md]: Updated. Thanks.

**Review by catalinaadam — APPROVED** [review]

**Pull request merged**